lib/commands/getViewportSize.js

```diff
@@ -21,7 +21,7 @@
  *
  * @alias browser.getViewportSize
  * @param {String} property  if "width" or "height" is set it returns only that property
- * @returns {Object}  viewport width and height of the browser
+ * @return {Object}  viewport width and height of the browser
```

Comment:
actually all of these getXXX command should have `{Type|Type[]}` as return type but this is a different story